lib/drives/pages/create_drive_page.dart

```diff
@@ -199,78 +199,66 @@ class _CreateDriveFormState extends State<CreateDriveForm> {
-                  child: Semantics(
-                    button: true,
-                    child: TextFormField(
-                      decoration: InputDecoration(
-                        border: const OutlineInputBorder(),
-                        labelText: S.of(context).formTime,
-                      ),
-                      readOnly: true,
-                      onTap: _showTimePicker,
-                      controller: _timeController,
-                      validator: _timeValidator,
+          const Divider(),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: <Widget>[
+              Expanded(
+                child: Semantics(
+                  button: true,
+                  child: TextFormField(
+                    decoration: InputDecoration(
+                      border: const OutlineInputBorder(),
+                      labelText: S.of(context).formDate,
                     ),
+                    readOnly: true,
+                    onTap: _showDatePicker,
+                    controller: _dateController,
                   ),
                 ),
-                const SizedBox(width: 50),
-                Expanded(
-                  child: SizedBox(
-                    //todo: add same height as time&date.
-                    height: 60,
-                    child: DropdownButtonFormField<int>(
-                      value: _dropdownValue,
-                      icon: const Icon(Icons.arrow_downward),
-                      decoration: InputDecoration(
-                        border: const OutlineInputBorder(),
-                        labelText: S.of(context).seats,
-                      ),
-                      onChanged: (int? value) {
-                        setState(() {
-                          _dropdownValue = value!;
-                        });
-                      },
-                      items: list.map<DropdownMenuItem<int>>((int value) {
-                        return DropdownMenuItem<int>(
-                          value: value,
-                          child: Text(value.toString()),
-                        );
-                      }).toList(),
+              ),
+              const SizedBox(width: 10),
+              Expanded(
+                child: Semantics(
+                  button: true,
+                  child: TextFormField(
+                    decoration: InputDecoration(
+                      border: const OutlineInputBorder(),
+                      labelText: S.of(context).formTime,
                     ),
+                    readOnly: true,
+                    onTap: _showTimePicker,
+                    controller: _timeController,
+                    validator: _timeValidator,
                   ),
                 ),
-              ],
+              ),
+            ],
+          ),
+          const Divider(),
+          SizedBox(
+            width: 150,
+            child: IncrementField(
```

Comment:
can we maybe add some kind of tooltip or also a Border around that so it's clear that you are selecting the seats.